sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/ParquetFooterReader.java

```diff
@@ -63,20 +61,10 @@ public static ParquetMetadata readFooter(
           .build()
           .getMetadataFilter();
     }
-    return readFooter(configuration, file.toPath(), filter);
+    return readFooter(HadoopInputFile.fromStatus(file.fileStatus(), configuration), filter);
```

Comment:
We can remove the `SKIP_ROW_GROUPS` and `WITH_ROW_GROUPS` now I think, as they are no longer used.